Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -244,60 +260,43 @@ public void onClick(View v) {
             @Override
             public void onClick(View view) {
                 JSONObject obj = new JSONObject();
-                try {
-                    obj.put("name", "test name");
-                    obj.put("auto_deeplink_key_1", "This is an auto deep linked value");
-                    obj.put("message", "hello there with short url");
-                    obj.put("$og_title", "this is new sharing title");
-                    obj.put("$og_description", "this is new sharing description");
-                    obj.put("$og_image_url", "https://s3-us-west-1.amazonaws.com/branchhost/mosaic_og.png");
-                } catch (JSONException ex) {
-                    ex.printStackTrace();
-                }
-
-                new Branch.ShareLinkBuilder(MainActivity.this, obj)
+                LinkProperties linkProperties = new LinkProperties()
+                        .addTag("myShareTag1")
+                        .addTag("myShareTag2")
+                         //.setAlias("mylinkName") // In case you need to white label your link
+                        .setChannel("myShareChannel2")
+                        .setFeature("mySharefeature2")
+                        .setStage("10")
+                        .addControlParameter("$android_deeplink_path", "custom/path/*")
+                        .addControlParameter("$android_deeplink_path", "custom/details/*")
```

Comment:
ditto -- use $android_url or some other control parameter for the second example